core/src/epicli/data/common/ansible/playbooks/roles/helm/tasks/main.yml

```diff
@@ -24,9 +22,9 @@
             helm search repo {{ helm_chart_repo_name }} --output json | jq -r '.[].name'
           register: helm_charts_list_before_update
 
-        - name: "Add {{ helm_chart_repo_name }} Helm repository from url {{ repository_url }}"
+        - name: "Add {{ helm_chart_repo_name }} Helm repository from url http://localhost"
           shell: |
-            helm repo add {{ helm_chart_repo_name }} {{ repository_url }}/helm-charts/system
+            helm repo add {{ helm_chart_repo_name }} http://localhost/helm-charts/system
```

Comment:
You sure http://localhost/helm-charts/system is correct? In my case I have http://localhost/epirepo/helm-charts/system/

Reply:
fixed